### DISCUSSION TITLE
findripples threshold error

### DISCUSSION THREAD

**Discussion by dlinsen1** [petersenpeter/CellExplorer#105] [Q&A] [answered]
Hello peter,

Thank you again for this nice tool and for your feedback to previous issues. We are trying different parameters for detecting HFOs using the GUI, but have been unable to use the std thresholding settings:
![image](https://github.com/petersenpeter/CellExplorer/assets/16261590/1a115356-aa55-4c6b-a239-4e9e1d8ee97d)
![image](https://github.com/petersenpeter/CellExplorer/assets/16261590/ec5ea053-cdad-47e5-9fdd-228b3e3c5876)
should we be entering these values differently? and how does checking the 'use absolute threshold' box impact things?

**Comment by petersenpeter** [comment, marked as answer]
Hi 

I can see this can be a little confusing. In both cases, the value has to be entered numerically. Either in units of standard deviation (e.g. 2) or in absolute units of µV (e.g. 18) The lower checkbox defines if the value is treated as in units of std or in absolute terms (µV).

**Reply by dlinsen1** [reply]
Worked like a charm unchecking box and entering numeric values thank you again.
